### ISSUE TITLE
the request './walletconnect' failed to resolve only because it was resolved as fully specified (probably because the origin is strict ecmascript module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"'

### ISSUE THREAD

**Issue by soldmysol** [solana-labs/wallet-adapter#308]
the request './walletconnect' failed to resolve only because it was resolved as fully specified (probably because the origin is strict ecmascript module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"'

I am getting this error when I run npm start, I tried nearly every single fix or update I could find in the last 12 hours but I am still stuck here , I even think updating some of my files messed this up even more, my package-lock.json file is now red ? I have no idea what is happening.

**Comment by jordansexton** [comment]
If you're filling out a bug report, please use the report template provided. It's there for a reason -- we can't see what your code is doing, so you need to provide detailed reproduction steps.

Also, the WalletConnect adapter is not released on npm. This is mentioned on the readme. If you're trying to use it, it's not going to work.

If you're building from source, please follow the steps here: https://github.com/solana-labs/wallet-adapter#build-from-source

**Comment by soldmysol** [comment]
> Collaborator

this is my first time posting on here sorry so for my issue do leave it on bug report?

**Comment by jordansexton** [comment]
Yes. But if you are trying to use WalletConnect, it won't work. Please read through the readme first on how to get started.

If that doesn't help, please open a new issue with the Bug Report template and fill it out. If you have a link to a repo that reproduces the issue that will help.

**Comment by jordansexton** [comment]
Try using this: https://github.com/solana-dev-adv/solana-dapp-next

It should work out of the box.

**Comment by soldmysol** [comment]
> 

that was one of the ones I tried to do earlier , sorry I have not coded or done coding related things in a few years but this is what returns to me upon trying  that 

PS C:\Users\paulg\Desktop\metaplex\js\packages\candy-machine-ui> npm run dev

> @solana/wallet-adapter-nextjs-starter@0.8.2 dev
> NODE_OPTIONS='--inspect' next dev

'NODE_OPTIONS' is not recognized as an internal or external command,
operable program or batch file.

**Comment by soldmysol** [comment]
> 

now when I try npm run I am getting this 

Error: Could not find a production build in the 'C:\Users\paulg\Desktop\metaplex\js\packages\candy-machine-ui\.next' directory. Try building your app with 'next build' before starting the production server. https://nextjs.org/docs/messages/production-start-no-build-id
    at Server.readBuildId (C:\Users\paulg\Desktop\metaplex\js\packages\node_modules\next\dist\server\next-server.js:1578:23)
    at new Server (C:\Users\paulg\Desktop\metaplex\js\packages\node_modules\next\dist\server\next-server.js:92:29)
    at NextServer.createServer (C:\Users\paulg\Desktop\metaplex\js\packages\node_modules\next\dist\server\next.js:107:16)
    at async C:\Users\paulg\Desktop\metaplex\js\packages\node_modules\next\dist\server\next.js:119:31
